setup.py

```diff
@@ -40,7 +40,7 @@ def readfile(filename, split=False):
     url='',
     license='APACHE',
     packages=find_packages(exclude=['ez_setup',]),
```

Comment:
This line should be:
`packages=find_packages('src'),`

Reply:
Will fix.

Reply:
You don't have it in mfxml2ex... what's the best usage here?
https://github.com/hsorby/mbfxml2ex/blob/master/setup.py uses:
`py_modules=["mbfxml2ex"],`
`package_dir={'': 'src'},`
Scaffoldmaker would have:
`packages=find_packages('src'),`
`package_dir={'': 'src'},`

Reply:
No mbfxml2ex doesn't have any packages so it really doesn't need to use the `find_packages` helper function.